app/controllers/admin/routes.py

```diff
@@ -27,15 +28,18 @@ def trackVolunteersPage(programID, eventID):
     if g.current_user.isCeltsAdmin:
         if ProgramEvent.get_or_none(ProgramEvent.event == eventID, ProgramEvent.program == programID):
             eventParticipantsData = EventParticipant.select().where(EventParticipant.event == eventID)
+            eventRsvpData = EventRsvp.select().where(EventRsvp.event == eventID)
+            eventRsvpUsers = []
             eventParticipantsData = eventParticipantsData.objects()
-
+            for data in eventRsvpData:
+                eventRsvpUsers.append(data.user)
```

Comment:
This extra step of changing to a list of users is unnecessary. You can just get user objects from the original query. This may also be the case with the event participants query.

Reply:
Yay! less code.